src/test/java/temporary/tests/BaseTest.java

```diff
@@ -24,4 +23,46 @@ public void baseSetUp() {
     public static void baseTearDown() {
         quitWebDriver();
     }
+
+    @Test
+    public void checkFacebookRedirect(){
+        // Given
+        openPage();
+        // When
+        clickOnFacebookButton();
+        // Then
+        var driver = getWebDriverInstance();
+        var tabs = driver.getWindowHandles();
+        Assert.assertEquals("New tab is not opened", 2, tabs.size());
+        validateNewTabURL("https://facebook.com/");
+    }
+
+    @Test
+    public void checkTwitterRedirect(){
+        // Given
+        openPage();
+        // When
+        clickOnTwitterButton();
+        // Then
+        var driver = getWebDriverInstance();
+        var tabs = driver.getWindowHandles();
+        Assert.assertEquals("New tab is not opened", 2, tabs.size());
+
+        validateNewTabURL("https://twitter.com/");
+    }
+
+    @Test
+    public void checkLinkedinRedirect(){
+        // Given
+        openPage();
+        // When
+        clickOnLinkedinButton();
+        // Then
+        var driver = getWebDriverInstance();
+        var tabs = driver.getWindowHandles();
+        Assert.assertEquals("New tab is not opened", 2, tabs.size());
+        validateNewTabURL("https://linkedin.com/");
+        // test nie przechodzi bo linkedin otwiera sie na tej samej karcie
```

Comment:
Delete comment 🙏

Reply:
You can check if test engine that you are using supports some way to mark tests that are failing :)